src/lib/deploy.js

```diff
@@ -241,17 +241,17 @@ async function deploy({ network, yes }) {
   const accountResponse = await sendGraphQL(graphQLEndpoint, accountQuery);
 
   let nonce = 0;
-  if (yes && !accountResponse?.data?.account?.nonce) {
-    // If running in non-interactive mode and no nonce is found, show an error message and return early
+  if (!accountResponse?.data?.account) {
+    // No account is found, show an error message and return early
     log(
       red(
-        `  Failed to find the account nonce.\n  Please run in interactive mode and specify an account nonce.`
+        `  Failed to find the specified account in the ledger using the specified URL.\n  Please make sure the account "${zkAppAddressBase58}" has previously been funded.`
```

Comment:
@MartinMinkov Any way to phrase this more clearly?

Reply:
Maybe ` Failed to find the fee payer's account on chain.\n Please make sure the account "${zkAppAddressBase58}" has previously been funded.` ?